.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Awesome seeing support for prettier! In an effort to keep improve traceability of the commit history, would you mind moving the addition of prettier to its own PR against #4 ? Also, might be good to add an npm script for running this in case users don't have it installed globally.